### PR TITLE
Implement optional daemons

### PR DESCRIPTION
**Pull request by stefanor**
A few of our services have some configuration-dependent daemons. Let's support blacklisting daemons.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: yodeploy/hooks/daemon.py [hunk 1]
```diff
@@ -32,10 +32,17 @@ def is_available(self):
 
     def install_daemons(self):
         for name, template, target in self.iter_templates():
-            log.info('Creating and restarting %s %s daemon', name, self.name)
-            self.app.template(template, target)
-            self.reload()
-            self.restart(name)
+            if self.app.is_daemon_enabled(name):
```
Comment (jamesodo) [inline yodeploy/hooks/daemon.py]: Where is the implementation of `is_daemon_enabled` that would return `False`?

Comment (stefanor) [inline yodeploy/hooks/daemon.py]: In the app.

**Pull request merged**